src/webapp/components/sidebar-nav/SidebarNavMenuGroup.tsx

```diff
@@ -19,18 +19,18 @@ interface SidebarNavProps {
 const SidebarNavMenuGroup: React.FC<SidebarNavProps> = ({ menu, groupName, className }) => {
     const classes = useStyles(menu.level);
 
-    const { module } = useGlassModuleContext();
+    const { currentModuleAccess } = useCurrentModuleContext();
 
     const isCurrent = (name: string) => {
-        return module ? module === name : false;
+        return currentModuleAccess ? currentModuleAccess.moduleName === name : false;
```

Comment:
It seems `currentModuleAccess` cannot be undefined.

Reply:
Addressed in https://github.com/EyeSeeTea/glass-dev/commit/c664c49dc8abcbeb0baeceac61d29b7d7275cb08